mbuild/utils/io.py

```diff
@@ -88,6 +88,9 @@ class DelayImportError(ImportError, SkipTest):
 
 # conda install -c conda-forge openbabel
 
+NOTE: openbabel is only available for python<3.13.
```

Comment:
Should we put this note in the environment.yml as well?

Reply:
Maybe just in the `environment-dev.yml` file since it's not a core dependency.